postprocessor/src/test/resources/partialupdate/StringOperationWithRemovedMemberGeneratedClient.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.autorest.postprocessor.util;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import fixtures.bodystring.implementation.StringOperationsImpl;
+
+/** Initializes a new instance of the synchronous AutoRestSwaggerBatService type. */
+@ServiceClient(builder = AutoRestSwaggerBatServiceBuilder.class)
+public final class StringOperationWithRemovedMemberGeneratedClient {
+
+    @Generated private final StringOperationsImpl serviceClient;
```

Comment:
nit: it would be better to have `@Generated` in a separate line. It's the pattern we use for `@JsonProperty` annotation too.

Reply:
In `ServiceSyncClientTemplate` (https://github.com/haolingdong-msft/autorest.java/blob/e4405a9c646d4a1306b9fb0eafd58049dfd42925/javagen/src/main/java/com/azure/autorest/template/ServiceSyncClientTemplate.java), we add `@Generated` for member variables **in a seperate line**. But after formatting, `@Generated` comes to the same line, it only happens to member variable declaration. So may need more investigation on 
 the formatter if we want `@Generated` to be in a seperate line.
```java
// Add service client member variable
      classBlock.annotation("Generated"); // This hold a seperate line
      if (wrapServiceClient) {
        classBlock.privateFinalMemberVariable(serviceClient.getClassName(), "serviceClient");
      } else {
        classBlock.privateFinalMemberVariable(methodGroupClient.getClassName(), "serviceClient");
      }
```

Reply:
@srnagar This is due to your javaformat. If you want this format, please update it.